pvlib/pvsystem.py

```diff
@@ -1580,15 +1580,21 @@ def sapm(effective_irradiance, temp_cell, module):
     Parameters
     ----------
     effective_irradiance : numeric
-        Effective irradiance (suns).
+        Effective irradiance [W/m^2].
 
     temp_cell : numeric
-        The cell temperature (degrees C).
+        Cell temperature [C].
 
     module : dict-like
         A dict or Series defining the SAPM parameters. See the notes section
         for more details.
 
+    irrad_ref : float (optional, default=1000)
```

Comment:
I prefer `reference_irradiance` as used below. I don't see an entry in the variables style guide.

Reply:
#825